lib/Doctrine/Query/Having.php

```diff
@@ -78,7 +78,7 @@ private function parseAggregateFunction($func)
      * @param mixed $value
      * @return string
      */
-    final private function _parseAliases($value)
```

Comment:
PHP 8 also complained about this. Since the function is private, it's not possible to be overwritten, so the `final` keyword here isn't necessary.